tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.PublicApiHasNotChanged.verified.txt

```diff
@@ -33,7 +33,6 @@ namespace Datadog.Trace.Configuration
         public int PartialFlushMinSpans { get; set; }
         public string TracesPipeName { get; set; }
         public int TracesPipeTimeoutMs { get; set; }
-        public string TracesUnixDomainSocketPath { get; set; }
```

Comment:
This shouldn't have gone I don't think 🤔

Reply:
This is correct, `TracesUnixDomainSocketPath` got removed from `ExporterSettings` here but it is still present on the `ImmutableExporterSettings` type if you expand below

Reply:
This was removed intentionlaly as it had been added only for .NET. (cf PR description)
This is the reason why Lucas rightfully asked me to document these settings (cf https://github.com/DataDog/architecture/pull/1000)